services/movie.py

```diff
@@ -1,37 +1,44 @@
+from django.db import transaction
 from django.db.models import QuerySet
+from django.shortcuts import get_object_or_404
 
+import init_django_orm  # noqa: F401
 from db.models import Movie
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
```

Comment:
Consider using `Optional[List[int]]` for `genres_ids` and `actors_ids` to explicitly indicate that these parameters can be `None`.